.github/workflows/pr.yml

```diff
@@ -70,29 +70,36 @@ jobs:
     uses: ./.github/workflows/production-build-without-database.yml
     secrets: inherit
 
+  integration-test:
+    name: Tests
+    needs: [changes, lint, build, build-api-v1, build-api-v2]
+    if: ${{ needs.changes.outputs.has-files-requiring-all-checks == 'true' }}
+    uses: ./.github/workflows/integration-tests.yml
+    secrets: inherit
+
   e2e:
-    name: E2E tests
+    name: Tests
```

Comment:
Naming all of these Tests results in the following grouping, which is nicer than having all of the tests separated.
![Screenshot 2024-05-17 at 3 02 56 PM](https://github.com/calcom/cal.com/assets/2538462/21d2df5d-0dea-4d65-abe5-bd6855289e05)